### PR TITLE
Landing page created

### PR DESCRIPTION
**Pull request by lape15**
image

### CODE REVIEW

**Comment by Nkemjiks** [comment]
@lape15 

I have made some updates to the PR to make the design closer to what we choose. As a front-end developer, making the page look exactly or close to the design you are given is very important. I added it as a commit so you can go through my changes. In other points..

- Please always remove any file you are not using before raising a PR. E.g. the images that were not used and some of the create react setup files that were not necessary
- Don't forgot to always remove commented out code
- Try to resolve any CLI issue like the variable that was defined but never used the landing.js file

File: src/Pages/Landing.js [hunk 1]
```diff
@@ -1,15 +1,48 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
 import { ReactComponent as ReactLogo } from "../assets/lady (1).svg";
+// import { ReactComponent as SmallLogo } from "../assets/lady.svg";
```
Comment (Nkemjiks) [inline src/Pages/Landing.js]: Remember to remove commented codes

File: src/Pages/Landing.js [hunk 2]
```diff
@@ -1,15 +1,48 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
```
Comment (Nkemjiks) [inline src/Pages/Landing.js]: Change all react file `js` extension file to `jsx`

File: src/assets/lady.svg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (Nkemjiks) [inline src/assets/lady.svg]: No need for any extra file you are not using. Always remember to delete them

File: src/Pages/Landing.js [hunk 4]
```diff
@@ -1,15 +1,48 @@
...
 import { ReactComponent as ReactLogo } from "../assets/lady (1).svg";
+// import { ReactComponent as SmallLogo } from "../assets/lady.svg";
+import { TimelineLite, Power3 } from "gsap";
```
Comment (Nkemjiks) [inline src/Pages/Landing.js]: Remember to add after all your import before doing other steps like defining variables for readability

File: src/styles/_landing.scss [hunk 5]
```diff
@@ -1,22 +1,100 @@
...
+      margin-top: 20px;
+      button {
+        border-radius: 30px;
+        background-color: transparent;
+        border: 1px solid #f9b500;
+        padding: 10px 15px;
+        letter-spacing: 1px;
+        font-weight: 300;
+        cursor: pointer;
+        // margin-top: 25px;
```
Comment (Nkemjiks) [inline src/styles/_landing.scss]: Commented code

File: src/styles/_landing.scss [hunk 6]
```diff
@@ -1,22 +1,100 @@
...
+      }
     }
   }
   .right {
-    border: 2px solid blue;
-    flex-basis: 50%;
+    flex-basis: 55%;
+    overflow: hidden;
+    @include mobile() {
+      //   width: 100%;
```
Comment (Nkemjiks) [inline src/styles/_landing.scss]: commented code

File: src/styles/_landing.scss [hunk 7]
```diff
@@ -1,22 +1,100 @@
...
+        margin: 0;
+        @include mobile() {
+          font-size: 1rem;
+          line-height: 28px;
+        }
+        .word {
+          opacity: 1;
+        }
+      }
+    }
```
Comment (Nkemjiks) [inline src/styles/_landing.scss:59]: Add a line spacing between style definition for readability

File: src/Components/Header.js [hunk 8]
```diff
@@ -4,11 +4,12 @@ const Header = () => {
   return (
     <header>
       <div className="left">
-        <svg width="100" height="100">
-          <circle
+        <svg width="120" height="120">
```
Comment (Nkemjiks) [inline src/Components/Header.js]: I have updated the logo image but you still haven't made it a link to the home page. I have marked that as incomplete on the board

**Review by Nkemjiks — CHANGES_REQUESTED** [review]
Please go through the review and make changes

**Comment by Nkemjiks** [comment]
Also, can you be adding a screenshot of your page to the PR description so i see what it looks like without having to run locally at first?

File: src/Pages/Landing.js (hunk 1 above)
Comment (lape15) [reply src/Pages/Landing.js]: Ohk

File: src/Pages/Landing.js (hunk 2 above)
Comment (lape15) [reply src/Pages/Landing.js]: Okay

> Also, can you be adding a screenshot of your page to the PR description so i see what it looks like without having to run locally at first?

I will do that

File: src/Pages/Landing.js (hunk 4 above)
Comment (lape15) [reply src/Pages/Landing.js]: Alright

File: src/Pages/Auth/Register.jsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (Nkemjiks) [inline src/Pages/Auth/Register.jsx]: Can you remove this file and the login component. Its not related to this card

**Comment by Nkemjiks** [comment]
You haven't changed the Header.js and Landing.js file extension to jsx

File: src/App.js [hunk 10]
```diff
@@ -1,13 +1,22 @@
...
+  useEffect(() => {
+    TweenMax.to(app, 0, { css: { visibility: "visible" } });
+  });
   return (
-    <div className="App">
+    <div className="App" ref={(el) => (app = el)}>
       <Header />
-      <Landing />
+      <Switch>
+        <Route exact path="/" component={Landing} />
```
Comment (Nkemjiks) [inline src/App.js:17]: Can your remove these routing too? Its not related to the card

File: src/Components/Header.js [hunk 11]
```diff
@@ -1,38 +1,21 @@
...
-              fontWeight: "800",
-              fontSize: "14px",
-              fontFamily: "Roboto sans-serif",
-            }}
-          >
-            BudgetIt
-          </text>
-        </svg>
+        <NavLink to="/">
+          {" "}
```
Comment (Nkemjiks) [inline src/Components/Header.js]: Why did you add this? I don't see its use

File: src/Components/Header.js [hunk 12]
```diff
@@ -1,38 +1,21 @@
...
+          <img src={logo} alt="BudgetIt logo" />{" "}
```
Comment (Nkemjiks) [inline src/Components/Header.js]: Same here

**Review by Nkemjiks — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 10 above)
Comment (lape15) [reply src/App.js:17]: > You haven't changed the Header.js and Landing.js file extension to jsx

I changed it, and  then it threw an error that bit could not find the file

File: src/Pages/Auth/Register.jsx (hunk 9 above)
Comment (lape15) [reply src/Pages/Auth/Register.jsx]: Ohk

**Comment by lape15** [comment]
screencapture-localhost-3002-2020-07-03-10_59_57

File: src/App.js (hunk 10 above)
Comment (Nkemjiks) [reply src/App.js:17]: You cancel and restart your server

File: src/Components/Header.jsx [hunk 13]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import logo from "../assets/budget-it.png";
+
+const Header = () => {
+  return (
+    <header>
+      <div className="left">
+        <NavLink to="/">
+          {" "}
```
Comment (Nkemjiks) [inline src/Components/Header.jsx]: This guy is still here

Comment (lape15) [reply src/Components/Header.jsx]: Trying to figure out where it comes from so I can disable it.
